.gitignore

```diff
@@ -24,6 +24,7 @@ images/upload/*
 testing/
 plugins/content_save_test/
 plugins/user_fields/
+plugins/custom_content_search_controls/

```

Comment:
should git ignore `plugins` and `plugins/**` and add the specific ones that come with the cms by default